nsv13/code/modules/munitions/ship_weapons/ballistic_weapons/broadsides.dm

```diff
@@ -166,3 +265,147 @@
 
 /obj/effect/particle_effect/muzzleflash/Destroy()
 	return ..()
+
+//Newer Broadsides features start here
+
+/obj/machinery/ship_weapon/broadside/MouseDrop_T(atom/movable/A, mob/user)
+	if(stovepipe)
+		to_chat(user, "<span class='warning'>The [src] is completely locked up, you have to <i>pry</i> out the stovepiped shell!</span>")
+		return FALSE
+	if(ismoth(A) || istype(A, /mob/living/simple_animal/mothroach && A != user))
+		mothclean(A, user)
+		return TRUE
+	if(prob(soot)) //likelihood of stovepipe is a lot higher if you crateload the gun
```

Comment:
This should be in the section that is only run if you are actually crateloading.